circle.yml

```diff
@@ -51,12 +51,5 @@ jobs:
           name: Build docker image and publish container tagged with branch
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS
-            bin/build.sh
-            docker push avvo/kafkamon:$CIRCLE_BRANCH
-      - run:
-          name: Publish latest if master
-          command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              docker tag avvo/kafkamon:release avvo/kafkamon:latest
-              docker push avvo/kafkamon:latest
```

Comment:
I'd like to keep `latest` as a tag, since that's referenced in the readme (and this is an open source project maybe someone else is using it?)

Reply:
Great point!  I will make sure this ks preserved

Reply:
Great point!  I will make sure this ks preserved